e2e/etcdctlv3_test.go

```diff
@@ -230,6 +253,70 @@ func getTest(cx ctlCtx) {
 		}
 	}
 }
+
+func txnTestSuccess(cx ctlCtx) {
+	defer close(cx.errc)
+
+	if err := ctlV3Put(cx, "sample", "value"); err != nil {
+		if cx.dialTimeout > 0 && isGRPCTimedout(err) {
+			cx.t.Fatalf("getTest error (%v)", err)
+		}
+	}
+
+	rqs := txnRequests{
+		compare: []request{
+			{reqType: versionCompReq, comp: "=", kv: kv{key: "sample", val: "1"}},
```

Comment:
Is there a benefit to using a request type? It really obscures what is being sent to the txn command-- `` ` ``version("sample") = "1" `` ` ``. I think plain strings would be better.